modules/giapi/src/main/scala/giapi/client/gpi/GPIClient.scala

```diff
@@ -10,12 +10,13 @@ import fs2.Stream
 import giapi.client.commands.{Command, CommandResult, Configuration}
 import giapi.client.{Giapi, commands}
 import mouse.boolean._
-import scala.concurrent.ExecutionContext
+import scala.concurrent.duration._
 
 /**
   * Client for GPI
   */
-class GPIClient[F[_]](giapi: Giapi[F], ec: ExecutionContext) {
+class GPIClient[F[_]](giapi: Giapi[F]) {
```

Comment:
now the `ExecutionContent` is passed to the giapi connection

Reply:
Is this used for blocking operations, or is it all nonblocking from the EPICs client point of view? This will affect the kind of EC you want to use here.